app/src/main-process/menu/build-default-menu.ts

```diff
@@ -315,7 +316,9 @@ export function buildDefaultMenu({
       separator,
       {
         id: 'view-repository-on-github',
-        label: __DARWIN__ ? 'View on GitHub' : '&View on GitHub',
+        label: __DARWIN__
+          ? `View ${displayViewInGitHub ? 'on GitHub' : 'in Browser'}`
+          : `&View ${displayViewInGitHub ? 'on GitHub' : 'in Browser'}`,
         accelerator: 'CmdOrCtrl+Shift+G',
         click: emit('view-repository-on-github'),
```

Comment:
```suggestion
        click: emit('view-repository-on-github'),
```

Should be updated to reflect it new purpose `view-repository-in-browser` (There will be a few places this needs updated besides here)